op-chain-ops/genesis/layer_one_test.go

```diff
@@ -78,10 +76,6 @@ func TestBuildL1DeveloperGenesis(t *testing.T) {
 	require.NoError(t, err)
 	require.EqualValues(t, 2, l2BlockTime.Uint64())
 
-	oracleAddr, err := portal.L2ORACLE(callOpts)
-	require.NoError(t, err)
-	require.EqualValues(t, deployments.L2OutputOracleProxy, oracleAddr)
```

Comment:
Can this be replaced with a call to `l2Oracle()` instead?

Reply:
it cannot be replaced due to the test being bad - it uses an old state dump that has old contract bytecode in it. We do have test coverage of this in chain assertions in solidity so tbh we should just delete this file because its fake test coverage